### PR TITLE
Add AFLOW to providers lists

### PR DESCRIPTION
**Pull request by ctoher**
Adds the directory providers/src/index-metadbs/aflow with info.json and links.json files. Also modifies the providers.json file in accordance with the instructions in the README.

### CODE REVIEW

**Review by CasperWA — COMMENTED** [review]
I don't see any issues.
However, do you not want to add a short description about AFLOW in the providers.json? It will be used in clients and such.

Also, your OPTIMADE database endpoint http://aflow.org/API/optimade/ seems to not supply entry listing or introspective endpoints. But I guess this is fine for now of course.

File: src/index-metadbs/aflow/v1/info.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : {
+      "attributes" : {
+         "available_api_versions" : [
+            {
+               "version" : "1.0.0",
+               "url" : "http://providers.optimade.org/index-metadbs/aflow/v1/"
```
Comment (ml-evs) [inline src/index-metadbs/aflow/v1/info.json:15]:
```suggestion
               "url" : "https://providers.optimade.org/index-metadbs/aflow/v1/"
```

File: src/links/v1/providers.json [hunk 2]
```diff
@@ -23,10 +23,10 @@
       "type": "links",
       "id": "aflow",
       "attributes": {
-        "name": "aflow.org",
+        "name": "AFLOW",
         "description": "",
-        "base_url": null,
-        "homepage": null,
+        "base_url": "http://providers.optimade.org/index-metadbs/aflow",
```
Comment (ml-evs) [inline src/links/v1/providers.json:28]:
```suggestion
        "base_url": "https://providers.optimade.org/index-metadbs/aflow",
```

**Review by ml-evs — COMMENTED** [review]
A couple of suggestions to use HTTPS below, feel free to ignore them. Will approve if requested for review again

**Comment by ctoher** [comment]
I've added a description of AFLOW to the providers.json, and changed http to https, as suggested.

**Review by CasperWA — APPROVED** [review]
Cheers.
Looking forward to seeing the implementation to also be reachable from the supplied URL :)

**Pull request merged**